app/src/main/java/org/mozilla/focus/fragment/UrlInputFragment.kt

```diff
@@ -760,12 +759,15 @@ class UrlInputFragment :
                 .remove(this)
                 .commit()
         } else {
-            val session = createTab(url, source = SessionState.Source.USER_ENTERED)
+            val tabId = requireComponents.tabsUseCases.addPrivateTab(
```

Comment:
If we use `addPrivateTab` here then we won't be able to apply/persist tracking protection exceptions? Maybe the `createTab` function or an alternative should go here?

Reply:
In the next PR I am changing `createTab()` to set  `private = true` and I would like to step by step get rid of `createTab()` and replace it by the use cases. This will temporarily break persisted TP exceptions until we get the GV patch, but it will get us closer to the state we want to be in.